server/router/router.go

```diff
@@ -57,13 +57,17 @@ func Load(noRouteHandler http.HandlerFunc, middleware ...gin.HandlerFunc) http.H
 	{
 		base.GET("/web-config.js", web.Config)
 
-		base.GET("/logout", api.GetLogout)
-		base.GET("/login", api.HandleLogin)
-		auth := base.Group("/authorize")
+		auth := base.Group("/auth")
 		{
-			auth.GET("", api.HandleAuth)
-			auth.POST("", api.HandleAuth)
-			auth.POST("/token", api.GetLoginToken)
+			auth.GET("/login", api.Auth)
+			auth.GET("/logout", api.GetLogout)
```

Comment:
Why this needs to be GET? From security perspective this is bad practice because you can redirect user to this endpoint to force his logout would be better to use endpoints:

GET /auth -> get token
DELETE /auth -> logout

Reply:
Oh, it's for redirects. Than it should be called GET /authorize and logout can be together with GET /token - to get token and DELETE /token - logout (call it as api and then push home route)

Reply:
If needed /logout can be the frontend route that can be redirected externally that would ask the user if he really wants to logout